backend/src/volunteerTables.js

```diff
@@ -0,0 +1,42 @@
+const express = require('express');
+const router = express.Router();
+
+const Volunteer = require("../models/Volunteer")
+const Hours = require("../models/hours")
+
+router.post('/', async (req, res) => {
+   const {email, key, data} = req.body
+
+   var query = {}
+   query[key] = data
+
+   Volunteer.updateOne({'email': email}, query).then(function(result) {
```

Comment:
Since we're using the volunteer's email as the primary key into the database, the site manager shouldn't be able to modify the email if the user has set up an account, since this will lock them out the next time they log in.